### ISSUE TITLE
Duplicating actors leads to issues with resulting duplicated items

### ISSUE THREAD

**Issue by apewall** [free-beer/bh2e#31]
Duplicating actors is a common usage case in order to quickly create similar actors.  Duplicated actors are provided a new unique actorId but all items retain the same itemId.  

If you attempt to delete or modify an item from a duplicated actor it will delete the first item found which matches the itemId instead of the item intended.

This seems to be caused by several functions in [shared.js](https://github.com/free-beer/bh2e/blob/master/module/shared.js) which assume the itemId will be unique when this is not the case.
- _deleteOwnedItem_
- _findItemFromId_
- _findActorFromItemId_

These should target the specific actor instead
for example _deleteOwnedItem_ could be replaced with:
`this.actor.deleteEmbeddedDocuments("Item", [element.dataset.id]);`

Which only deletes the item of the actor which is the current target.

**Comment by free-beer** [comment]
Yeah, I know about this one. To be honest it could be argued that the issue here is with Foundry in that they are only doing a shallow copy. I'd also add that generally this primarily applies to creatures (the documentation points out that you should link characters back to their original actor data or you'll experience issues anyway). I'll have a think about this one as I'm not sure it's urgent (unless you can make a case to the contrary?).
